locale/en.yml

```diff
@@ -839,6 +839,11 @@ en:
       ManageIQ::Providers::StorageManager:                                  Storage Manager
       ManageIQ::Providers::StorageManager::CinderManager:                   Storage Manager (Cinder)
       ManageIQ::Providers::StorageManager::SwiftManager:                    Storage Manager (Swift)
+      ManageIQ::Providers::Microsoft::InfraManager::Host:                   Host (Microsoft)
+      ManageIQ::Providers::Openstack::InfraManager::Host:                   Host (OpenStack)
+      ManageIQ::Providers::Redhat::InfraManager::Host:                      Host (Redhat)
+      ManageIQ::Providers::Vmware::InfraManager::Host:                      Host (Vmware)
+      ManageIQ::Providers::Vmware::InfraManager::HostEsx:                   Host (Vmware)
```

Comment:
Well in reality all VMware hosts are of type `ManageIQ::Providers::Vmware::InfraManager::HostEsx` since that includes `ESX` and `ESXi`.  Those are the only two types of `vSphere` hosts that we're going to encounter.  The problem is `Vmware::InfraManager::HostEsx` includes both `ESX` and `ESXi`, so saying `Vmware ESX Host` is confusing because 99% of the time it will be an `ESXi` host.

I'm all for staying consistent though.